src/mumble/Themes.cpp

```diff
@@ -144,5 +144,6 @@ bool Themes::readStylesheet(const QString &stylesheetFn, QString &stylesheetCont
 
 QString Themes::getDefaultStylesheet() {
 	return QLatin1String(".log-channel{text-decoration:none;}.log-user{text-decoration:none;}p{margin:0;}#qwMacWarning,"
-						 "#qwInlineNotice{background-color:#FFFEDC;border-radius:5px;border:1px solid #B5B59E;}");
+						 "#qwInlineNotice{background-color:#FFFEDC;border-radius:5px;border:1px solid #B5B59E;}"
+						 "#qwMacWarning > QLabel,#qwInlineNotice > QLabel{color:#333;}");
```

Comment:
Isn't `qwMacWarning` a macOS-specific element? What it its purpose here? :thinking:

Reply:
Not sure what you mean :thinking: Both ``#qwInlineNotice`` and ``#qwMacWarning`` have that faint yellow background color set. The ``#qwMacWarning`` is in line 146 and the ``,`` means both elements will have the following css rules applied.
So the ``#qwMacWarning > QLabel,#qwInlineNotice > QLabel`` also applies the dark text to both warnings.

The mac warning was ported from an inline stylesheet in 613b7fc6f3ce

Reply:
Ah woops - my CSS-foo was weak today :sweat_smile: